src/constants/fieldTypes.js

```diff
@@ -85,22 +84,6 @@ export const fieldTypeChoices = [
     backendType: backendTypes.individual,
     backendMultiple: false,
   },
-  {
-    labelId: 'RELATIONSHIPS_SELECTOR',
-    value: fieldTypes.relationships,
-    defaultValue: [],
-    backendType: backendTypes.string,
-    backendMultiple: true,
-    configuration: [
-      {
-        labelId: 'RELATIONSHIP_OPTIONS',
-        descriptionId: 'RELATIONSHIP_OPTIONS_HELPER_TEXT',
```

Comment:
Can these also be removed from `en.json`?

Reply:
looks like they can!